samples/wifi/shell/overlay-tfm.conf

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_NET_STATISTICS=n
+CONFIG_NET_STATISTICS_WIFI=n
+CONFIG_NET_STATISTICS_USER_API=n
+
+CONFIG_NET_PKT_RX_COUNT=8
+CONFIG_NET_PKT_TX_COUNT=4
+
+# Below section is the primary contributor to SRAM and is currently
+# tuned for performance, but this will be revisited in the future.
+CONFIG_NET_BUF_RX_COUNT=8
+CONFIG_NET_BUF_TX_COUNT=8
+CONFIG_NET_BUF_DATA_SIZE=128
+CONFIG_HEAP_MEM_POOL_SIZE=153600
+CONFIG_NET_TC_TX_COUNT=1
+
+CONFIG_NET_TX_STACK_SIZE=2048
+CONFIG_NET_RX_STACK_SIZE=2048
+
+# printing of scan results puts pressure on queues in new locking
+# design in net_mgmt. So, use a typical number of networks in a crowded
+# environment as the queue depth.
+CONFIG_NET_MGMT_EVENT_QUEUE_SIZE=10
+
+CONFIG_BUILD_WITH_TFM=y
+CONFIG_TFM_PROFILE_TYPE_NOT_SET=y
+CONFIG_TFM_BL2=n
+CONFIG_ENTROPY_PSA_CRYPTO_RNG=y
+CONFIG_MBEDTLS_PSA_CRYPTO_C=y
+CONFIG_MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG=y
```

Comment:
This ends up as disabled.

Conflicting with this?
```
# Needed for internal entropy
config MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
    default n if WPA_SUPP_CRYPTO
```

Reply:
Yeah, removed this now.